showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITPagination.java

```diff
@@ -0,0 +1,146 @@
+import com.google.showcase.v1beta1.it.util.TestClientInitializer;
+import java.util.concurrent.TimeUnit;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITPagination {
+
+  private static EchoClient grpcClient;
+  private static EchoClient httpjsonClient;
+
+  @BeforeClass
+  public static void createClients() throws Exception {
+    // Create gRPC Echo Client
+    grpcClient = TestClientInitializer.createGrpcEchoClient();
+    // Create Http JSON Echo Client
+    httpjsonClient = TestClientInitializer.createHttpJsonEchoClient();
+  }
+
+  @AfterClass
+  public static void destroyClients() throws InterruptedException {
+    grpcClient.close();
+    grpcClient.awaitTermination(TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+    httpjsonClient.close();
+    httpjsonClient.awaitTermination(
+        TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
+  }
+
+  // This tests that pagination returns the correct number of pages + responses and that
+  // the content is correct.
+  //
+  // The pageToken is where the streaming responses come back from and the page size denotes
+  // how many of the responses come back together. i.e for PageSize = 2 and PageToken = 1, see
+  // below:
+  //         | A | Series  | Of  | Words | That  | Will  | Be  | Sent  | Back  | One | By  | One
+  // Page #  | X | 1       | 1   | 2     | 2     | 3     | 3   | 4     | 4     | 5   | 5   | 6
+  // Token # | 0 | 1       | 2   | 3     | 4     | 5     | 6   | 7     | 8     | 9   | 10  | 11
+  @Test
+  public void testPagedExpandWithTokenGrpc() {
+    int pageSize = 2;
+    int pageToken = 1;
+    String content = "A series of words that will be sent back one by one";
+    int contentLength = content.split(" ").length;
+
+    EchoClient.PagedExpandPagedResponse pagedExpandPagedResponse =
+        grpcClient.pagedExpand(
+            PagedExpandRequest.newBuilder()
+                .setContent(content)
+                .setPageSize(pageSize)
+                .setPageToken(String.valueOf(pageToken))
+                .build());
+
+    String[] expected = content.split(" ");
+    int numExpectedPages = ((contentLength - pageToken) / pageSize);
+    // If the responses can't be evenly split into pages, then the extra responses
+    // will go to an additional page
+    if ((contentLength - pageToken) % pageSize != 0) {
+      numExpectedPages++;
+    }
+    int numExpectedResponses = contentLength - pageToken;
+
+    validatePagedResponses(
+        pagedExpandPagedResponse, expected, pageToken, numExpectedResponses, numExpectedPages);
+  }
+
+  // This tests that pagination returns the correct number of pages + responses and that
+  // the content is correct.
+  //
+  // The pageToken is where the streaming responses come back from and the page size denotes
+  // how many of the responses come back together. i.e for PageSize = 2 and PageToken = 1, see
```

Comment:
Do you mind making the PageToken a larger value like `3` to make it clearer? Maybe it's just me but it took me a while to figure out that what X means, I thought `X` and `0` are a pair ;)

Reply:
Sure, will do! I'll change it to `-` and hopefully that'll be clearer for others too.